MapboxNavigationTests/InstructionsBannerViewSnapshotTests.swift

```diff
@@ -275,13 +276,34 @@ class InstructionsBannerViewSnapshotTests: FBSnapshotTestCase {
         view.set(makeVisualInstruction(.takeOffRamp, .right, primaryInstruction: primary, secondaryInstruction: [secondary]))
         verifyView(view, size: view.bounds.size)
     }
+    
+    func testGenericShields() {
+        let window = UIApplication.shared.delegate!.window!!
+        let view = instructionsView()
+        styleInstructionsView(view)
+        view.maneuverView.isStart = true
+        view.distance = 482
+        
+        let primary = [
+            VisualInstructionComponent(type: .image, text: "ANK 1", imageURL: nil, abbreviation: nil, abbreviationPriority: NSNotFound),
+            VisualInstructionComponent(type: .text, text: "Ankh-Morpork 1", imageURL: nil, abbreviation: nil, abbreviationPriority: NSNotFound)
+        ]
+        
+        let secondary = [VisualInstructionComponent(type: .text, text: "Vetinari Way", imageURL: nil, abbreviation: nil, abbreviationPriority: NSNotFound)]
+        
+        window.addSubview(view)
+        DayStyle().apply()
+        
+        view.set(makeVisualInstruction(.reachFork, .right, primaryInstruction: primary, secondaryInstruction: secondary))
+        verifyView(view, size: view.bounds.size)
+    }
 }
 
 extension InstructionsBannerViewSnapshotTests {
     
-    func verifyView(_ view: UIView, size: CGSize) {
+    func verifyView(_ view: UIView, size: CGSize, tolerance: CGFloat = 0.01) {
         view.frame.size = size
-        FBSnapshotVerifyView(view, tolerance: 0.01)
+        FBSnapshotVerifyView(view, suffixes: ["_64"], tolerance: tolerance)
```

Comment:
This looks suspect.

Reply:
The `suffixes: ["_64"]`? That just tells the snapshot framework that we only have reference images in the `ReferenceImages_64` folder, so it doesn't try to look in the `ReferenceImages_32` folder as a fallback. (Default Behavior)

This basically removes unneeded (since we don't do 32-bit testing) noise from the output log.